sail/src/main/java/org/apache/rya/rdftriplestore/inference/InferenceEngine.java

```diff
@@ -73,6 +74,8 @@
     private Set<URI> transitivePropertySet;
     private Map<Resource, Map<URI, Value>> hasValueByType;
     private Map<URI, Map<Resource, Value>> hasValueByProperty;
+    private Map<URI, Set<URI>> domainByType;
```

Comment:
Can one of you create a ticket for this?  The solution of this is probably not within the scope of this task, but this should be resolved.